Configuration/NodeTypes.Overrides.yaml

```diff
@@ -0,0 +1,9 @@
+'Neos.Neos:Node':
+  options:
+    fusion:
+      prototypeGenerator: ~
```

Comment:
Hey, if you check the generator I linked, it generates prototypes both for `Acme.Demo:Page` and `Acme.Demo:Page.Document`.

Why it's hard to explain?
Because it's the first damn thing that people encounter. Because of this magic, people often don't distinguish between prototypes and nodetypes. You have to tell them, look, these are nodetypes, now for every nodetype depending on its type there is some invisible Fusion code generated, that you can't really see, but it's there. To understand where it's coming from, look at this PHP service here and... You get the point.
I'm strongly convinced, that for beginners the ease of understanding comes before the convenience of typing less. It's not just my opinion, I was surprised by it, but that's the feedback I got from all newbies making their first steps.
As said, if you guys would insist on keeping the magic on by default, let's at least adjust it to something meaningful.